src/entrypoints/custom-panel.ts

```diff
@@ -1,5 +1,6 @@
 // Compat needs to be first import
 import "../resources/compatibility";
+import { setCancelSyntheticClickEvents } from "@polymer/polymer/lib/utils/settings";
```

Comment:
We don't need it here, as we don't load Polymer code ourself

Reply:
I wasn't sure about putting it there, but I assumed it was needed since polymer was imported a few lines down.